components/cam/src/physics/crm/openacc_utils.F90

```diff
@@ -8,7 +8,7 @@ module openacc_utils
   use cudafor
 #endif
   implicit none
-  integer :: ierr
+  integer, private :: ierr
```

Comment:
Usually prefer to see "private" by default for everything in the module, and then explicitly make certain variables/routines public. Is there a reason to instead assume everything is public here?

Reply:
Yes all modules should have "private" as default.

Reply:
addressed in recent commit